src/common/decorators/exclude-from-query.decorator.ts

```
@@ -0,0 +1,11 @@
+import { SetMetadata } from '@nestjs/common';
```

ESLint

[error] 'SetMetadata' is defined but never used.

(@typescript-eslint/no-unused-vars)